pkg/logs/pipeline/provider.go

```diff
@@ -181,6 +183,17 @@ func (p *provider) NextPipelineChan() chan *message.Message {
 	return nextPipeline.InputChan
 }
 
+// NextPipelineChanWithMonitor returns the next pipeline input channel with it's monitor.
+func (p *provider) NextPipelineChanWithMonitor() (chan *message.Message, metrics.PipelineMonitor) {
```

Comment:
I left the old call site around because other parts of the code base use the logs pipeline but not for logs (epforwarder). 

I should call this out in the PR description but this PR only attempts to measure the file tailer for now. 
I was planning to address the other cases in a followup PR once we get some real world testing on this data.